OpenRA.Mods.Common/Traits/World/Locomotor.cs

```diff
@@ -432,8 +437,15 @@ void UpdateCellCost(CPos cell)
 				cost = terrainInfos[index].Cost;
 
 			var cache = cellsCost[cell.Layer];
-
-			cache[cell] = cost;
+			if (CellCostChanged is null)
```

Comment:
I don't think we're using `is` anywhere else in the codebase, so `== null` for consistency?

Reply:
Updated